.github/workflows/python-app.yml

```diff
@@ -9,25 +9,36 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.8', '3.10', '3.12']
```

Comment:
@Myrausman we can have all the current supported versions of `Python 3.x` and not the ones which are no longer supported.